opengrok-indexer/src/main/java/org/opengrok/indexer/history/Annotation.java

```diff
@@ -91,6 +92,15 @@ public String getRevision(int line) {
         return ret;
     }
 
+    @VisibleForTesting
+    Set<String> getAuthors() {
+        Set<String> ret = new HashSet<>();
```

Comment:
I personally like the concise:
```java
this.lines.stream().map(ln -> ln.author).collect(Collectors.toSet());
```

Reply:
Yeah, I was thinking about using stream() and then forgot so thanks. It definitely looks better.